### PR TITLE
ChipScreen - adding example for dynamic label

### PR DESCRIPTION
**Pull request by Inbal-Tish**
ChipScreen - adding example for onPress to open Dialog and dynamic label.
Chip - moving default values from static.
WheelPickerDialog - container self-alignment and code cleanup.

## Description
ChipScreen - adding example for dynamic label

## Changelog
ChipScreen - adding example for onPress to open Dialog and dynamic label.

### CODE REVIEW

File: demo/src/screens/componentScreens/ChipScreen.tsx [hunk 1]
```diff
@@ -1,13 +1,90 @@
...
+        {color.label}
+      </Text>
+    );
+  };
+
+  renderContent = () => {
+    const {selectedValue} = this.state;
+
+    return (
+      <WheelPickerDialog
```
Comment (ethanshar) [inline demo/src/screens/componentScreens/ChipScreen.tsx:60]: Why did you add a picker to the chip example screen? 
Don't you think it's making the example screen a bit more complicated than needed? 

What do you want to demonstrate here?

Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/ChipScreen.tsx:60]: That was a request by ux

Comment (ethanshar) [reply demo/src/screens/componentScreens/ChipScreen.tsx:60]: ohh ok :/

**Review by ethanshar — APPROVED** [review]
